Sources/_CryptoExtras/AES/Nonces.swift

```diff
@@ -0,0 +1,298 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftCrypto open source project
+//
+// Copyright (c) 2024 Apple Inc. and the SwiftCrypto project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.md for the list of SwiftCrypto project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Crypto
+import Foundation
+// MARK: - Generated file, do NOT edit
+// any edits of this file WILL be overwritten and thus discarded
+// see section `gyb` in `README` for details.
+
+// MARK: - AES._CBC + IV
+extension AES._CBC {
+    /// A value used once during a cryptographic operation and then discarded.
+    ///
+    /// Don’t reuse the same nonce for multiple calls to encryption APIs. It’s critical
+    /// that nonces are unique per call to encryption APIs in order to protect the
+    /// integrity of the encryption.
+    public struct IV: Sendable, ContiguousBytes, Sequence {
+        typealias IVTuple = (
+            UInt8, UInt8, UInt8, UInt8, UInt8, UInt8, UInt8, UInt8, 
+            UInt8, UInt8, UInt8, UInt8, UInt8, UInt8, UInt8, UInt8
+        )
+    
+        private var bytes: IVTuple
+
+        /// Creates a new random nonce.
+        public init() {
+            var bytes = IVTuple(0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0)
```

Comment:
I suspect the missing spaces after commas here will trigger the format checker.

Reply:
:doh:  I'll get that

Reply:
I've updated this, and made a slight tweak to add a "blankBytes" variable to not have to copy the 0's in a few different places.